<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferBalanceView</class>
 <widget class="QWidget" name="ReferBalanceView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OctoVault VPN</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 15px;
color:#FACA7F;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QWidget" name="balanceHolder" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
}

QWidget:hover{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 10px;
}</string>
   </property>
   <widget class="QLabel" name="lblBalance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:#818083;
border-color:transparent;
}</string>
    </property>
    <property name="text">
     <string>$0.00</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblBalanceHeader">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#818083;
}</string>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWithdraw">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
color:#818083;
}</string>
   </property>
   <property name="text">
    <string>WITHDRAW</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblReferHeader">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#818083;
}</string>
   </property>
   <property name="text">
    <string>Referral code</string>
   </property>
  </widget>
  <widget class="QWidget" name="ReferHolder" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border-color:transparent;
border: 1px solid #36353a;
border-radius: 10px;
}

QWidget:hover{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 10px;
}</string>
   </property>
   <widget class="QLabel" name="lblRefer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:#FFFFFF;
border-color:transparent;
}</string>
    </property>
    <property name="text">
     <string>$0.00</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnCopy">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
color:#818083;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/img/images/icon_copy.png</normaloff>:/img/images/icon_copy.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShare">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
color:#818083;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../assets/assets.qrc">
     <normaloff>:/img/images/icon_share.png</normaloff>:/img/images/icon_share.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lblRedeemHeader">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#818083;
}</string>
   </property>
   <property name="text">
    <string>Redeem your invite code</string>
   </property>
  </widget>
  <widget class="QWidget" name="redeemHolder" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border-color:transparent;
border: 1px solid #36353a;
border-radius: 10px;
}

QWidget:hover{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 10px;
}</string>
   </property>
   <widget class="QLineEdit" name="editRedeem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background:transparent;
padding-left:0px;
color: #36353a;
border-color: transparent;
}

QLineEdit:hover{
background:transparent;
padding-left:0px;
color: #FACA7F;
border-color: transparent;
}</string>
    </property>
    <property name="placeholderText">
     <string>Enter Refer Code</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnApply">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
color:#818083;
}</string>
   </property>
   <property name="text">
    <string>APPLY</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSpinner">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>300</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
